apps/web/src/components/V2/AlertV2.tsx

```diff
@@ -0,0 +1,112 @@
+import { motion, AnimatePresence } from "framer-motion";
```

Comment:
this can just be called Alert since its already under v2 directory
```
// apps/web/src/components/v2/alert.tsx
...
export const Alert = () => ...
```

we are going to change all filenames to kebab spacing instead of pascal case